ocs_ci/helpers/dr_helpers_ui.py

```diff
@@ -717,8 +707,7 @@ def verify_application_present_in_ui(acm_obj, workloads_to_check=[], timeout=60)
 
     """
     if workloads_to_check:
-        ocp_version = get_ocp_version()
-        acm_loc = locators[ocp_version]["acm_page"]
+        acm_loc = locators_for_current_ocp_version["acm_page"]
```

Comment:
shouldn't it be used like this `locators_for_current_ocp_version()["acm_page"]` ?

Reply:
it should ;), fixed everywhere